modules/account-lib/package.json

```diff
@@ -42,6 +42,7 @@
     "@bitgo/sdk-coin-hash": "^1.4.11",
     "@bitgo/sdk-coin-hbar": "^1.5.11",
     "@bitgo/sdk-coin-injective": "^1.4.11",
+    "@bitgo/sdk-coin-islm": "^1.0.0",
```

Comment:
Unless there is a specific use case for using this in account-lib elsewhere, let's remove it.

Reply:
It's needed for wallet-platform